test/boost/secondary_index_test.cc

```diff
@@ -649,20 +663,6 @@ SEASTAR_TEST_CASE(test_secondary_index_collections) {
         res = e.execute_cql("SELECT p from t where l1 CONTAINS 1").get0();
         assert_that(res).is_rows().with_size(0);
 
-        BOOST_REQUIRE_EXCEPTION(e.execute_cql("create index on t(         s2 )").get(), ire, non_full);
-        e.execute_cql(                        "create index on t(FULL    (s2))").get();
-        BOOST_REQUIRE_EXCEPTION(e.execute_cql("create index on t(FULL    (s2))").get(), ire, duplicate);
-
-        BOOST_REQUIRE_EXCEPTION(e.execute_cql("create index on t(         m2 )").get(), ire, non_full);
-        e.execute_cql(                        "create index on t(FULL    (m2))").get();
-        BOOST_REQUIRE_EXCEPTION(e.execute_cql("create index on t(FULL    (m2))").get(), ire, duplicate);
-
-        BOOST_REQUIRE_EXCEPTION(e.execute_cql("create index on t(         l2 )").get(), ire, non_full);
-        e.execute_cql(                        "create index on t(FULL    (l2))").get();
-        BOOST_REQUIRE_EXCEPTION(e.execute_cql("create index on t(FULL    (l2))").get(), ire, duplicate);
-
-        BOOST_REQUIRE_EXCEPTION(e.execute_cql("select * from t where m2[1] = '1'").get(), ire, entry_eq);
-
         res = e.execute_cql("SELECT p from t where s2 = {2}").get0();
         assert_that(res).is_rows().with_rows({{{int32_type->decompose(2)}}});
```

Comment:
I think that an eventually() is also needed.

Reply:
It's not needed. Because this is a single-node test, the view replica and the base replica are the same node, and the view update is, since a couple of years ago, guaranteed to be synchronous (the base write waits also for the view write). So we don't need eventually() in tests like this.
This is also true in dtests in some configurations (e.g., N=3 and RF=3) but other configurations (e.g., N>3) still need the eventually thing.